spot_driver/src/api/default_robot_state_client.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2023 Boston Dynamics AI Institute LLC. All rights reserved.
+
+#include <bosdyn/api/robot_state.pb.h>
+#include <spot_driver/api/default_robot_state_client.hpp>
+#include <spot_driver/api/time_sync_api.hpp>
+#include <spot_driver/conversions/geometry.hpp>
+#include <spot_driver/conversions/robot_state.hpp>
+
+namespace spot_ros2 {
+
+DefaultRobotStateClient::DefaultRobotStateClient(::bosdyn::client::RobotStateClient* client,
```

Comment:
I think we need consistency in the name. Either we change `_client` to `_api` or we change the name of the `kinematic_api` into `kinematic_client`.

Reply:
As I mentioned [here](https://github.com/bdaiinstitute/spot_ros2/pull/209#discussion_r1484902853), I'll make a separate PR to make our naming convention consistent.